src/views/Users/Login.vue

```diff
@@ -3,21 +3,158 @@
     id="loginPage"
     class="login"
   >
-    <h2>The login page will be here</h2>
-    <div>
-      <router-link to="/accounts/signup">
-        Register
-      </router-link>
-    </div>
+    <v-container fluid>
+      <!-- tabulation controls -->
+      <v-row>
+        <v-col
+          cols="12"
+        >
+          <v-btn class="mr-5"
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: ['class' should be on a new line.](https://app.codacy.com/gh/FAIRsharing/fairsharing.github.io/pullRequest?prid=5156911)